jobs/build_docker/docker_post_test.groovy

```diff
@@ -63,6 +65,23 @@ def generateTestBranches(function_test){
                                         env.DOCKER_PATH="${env.DOCKER_STASH_PATH}"
                                         env.DOCKER_RECORD_PATH="${env.DOCKER_RECORD_STASH_PATH}"
                                     }
+
+                                    // Add commit/version checking in docker-post-test
+                                    unstash "$DOCKER_STASH_NAME"
+                                    env.DOCKER_PATH="${env.DOCKER_STASH_PATH}"
+                                    env.DOCKER_RECORD_PATH="${env.DOCKER_RECORD_STASH_PATH}"
+                                    println "[DEBUG] stash_manifest_name:" + "$stash_manifest_name"
+                                    unstash "$stash_manifest_name"
+                                    env.MANIFEST_FILE="$stash_manifest_path"
+                                    
+                                    env.DOCKER_REPO_HASHCODE_FILE = env.RackHD_DIR + "/docker/docker_repo_hashcode.txt"
+                                    sh "/bin/bash ./build-config/jobs/build_docker/get_docker_commit_version.sh"
+                                    sh '''#!/bin/bash
+                                    ./build-config/build-release-tools/HWIMO-BUILD ./build-config/build-release-tools/application/docker_version_check.py \
+                                    --manifest-file $MANIFEST_FILE \
+                                    --parameters-file $DOCKER_REPO_HASHCODE_FILE
+                                    '''
+
```

Comment:
The steps of the PR:
1.```get_docker_commit_version.sh``` loads docker images and runs them
2. ```prepare_common.sh``` stop the running docker containers.
3. ```prepare_docker_post_test.sh``` loads docker images and runs them again

I think it's better to run the steps as below:
1. ```prepare_common.sh``` stop the running docker containers.
2. ```prepare_docker_post_test.sh``` loads docker images and runs them.
3. ```get_docker_commit_version.sh``` gets commit string from running containers by ```docker exec ...```

Also, running the script ```get_docker_commit_version.sh``` to run docker  without cleaning environment may cause error if there are running containers